src/main/java/com/meraphorce/services/UserService.java

```diff
@@ -106,21 +92,19 @@ public UserDTO getUserById(String idValue) {
      * @author Juan Chavez
      * @since May/14/2024
      */
-    public UserDTO updateUser(String idValue, UserDTO userUpdated) {
-    	UserDTO user = null;
+    public User updateUser(String idValue, User userUpdated) {
     	User data = null;
```

Comment:
The name "data" of the identifier is ambiguous and does not have context, try to add more descriptive variable
userUpdated, userSaved, etc